model/detector.py

```diff
@@ -0,0 +1,35 @@
+import cv2
+import numpy as np
+from PIL import Image
+
+class TextDetector:
+    def resize_image(self, image: Image.Image, max_size: tuple = (2000, 2000)) -> tuple:
```

Comment:
You can use return type hint like this:

from typing import Tuple

def resize_image(...) -> Tuple[Image, float]: